code/game/objects/structures/barsign.dm

```diff
@@ -44,10 +44,6 @@
 	if(!broken && !(flags & NODECONSTRUCT))
 		broken = TRUE
 
-/obj/structure/sign/barsign/deconstruct(disassembled = TRUE)
-	new /obj/item/stack/sheet/metal(drop_location(), 2)
-	new /obj/item/stack/cable_coil(drop_location(), 2)
-	qdel(src)
```

Comment:
Should remain here. This is called when you destroy the sign which makes it drop the items listed above